src/applications/disability-benefits/all-claims/tests/pages/reservesNationalGuardService.unit.spec.jsx

```diff
@@ -1,87 +0,0 @@
-import React from 'react';
-import { expect } from 'chai';
-import sinon from 'sinon';
-import {
-  DefinitionTester,
-  fillData,
-  fillDate,
-} from 'platform/testing/unit/schemaform-utils.jsx';
-import { mount } from 'enzyme';
-import formConfig from '../../config/form';
-
-describe('Reserve information', () => {
```

Comment:
I did, and I could _swear_ I wrote an explanation, but now I can't find it, if I did, so thank you for asking! It turns out (through Slack discussion sleuthing) that traditional unit tests like this are incompatible with the new VA web components since the interactive bits are in a shadow DOM and that's inaccessible to the test. I spent some time trying to salvage this test, but I couldn't figure anything out. I think we need to rely on the Cypress tests for this.

Reply:
I'll leave this open for a bit in case you have any insight! Let me know if you see anything I don't. 👀

Reply:
Ah, that's helpful to know! Thanks, that sounds good to me. 👍🏻